ecosystem/sep-0038.md

```diff
@@ -0,0 +1,219 @@
+## Preamble
+
+```
+SEP: 0038
+Title: Request for Quote API
```

Comment:
I think the title should qualify this as an Anchor specific API. Not to be confused with an OTC style RFQ (on-chain<>on-chain) API.
Perhaps `Anchor RFQ API`? The RFQ abbreviation is fairly known term.